pkg/network/peer.go

```diff
@@ -40,14 +46,14 @@ type Peer interface {
 	// done once for all peers). It does nothing if the peer has not yet
 	// completed handshaking. This queue is intended to be used for unicast
```

Comment:
s/`It does nothing`/`It returns an error `.

Reply:
Fixed.